### DISCUSSION TITLE
Is koreader (android version) recommended on non e ink screens like LCD, etc?

### DISCUSSION THREAD

**Discussion by wa15** [koreader/koreader#7766] [Q&A] [answered]
I dont know much about e ink but i read they refresh the screen a lot. I use librera and the battery usage is ~0%. while koreader battery usage was ~13%.
Are there specific e ink settings/themes i need to change in koreader to lower its battery usage on my amoled tablet? or is koreader (android) just designed for e ink readers that run android?
thanks

**Comment by Frenzie** [comment, marked as answer]
Doesn't the scenario you describe mean you should enable night mode to keep most of the screen black (if you want to save battery)?

You've got the screen refreshing thing backwards, btw. E Ink is static; it can stay that way "forever," even without any power. It's a regular screen that refreshes 60 times per second or more.

**Reply by wa15** [reply]
I was using both under the same condition, black text on white background. I was curious since librera has an "e-ink setting" so i was wondering if koreader had an "lcd" setting. (e-ink setting on librera acutally changes alot). so it comes back to my original question on whether the developer of koreader intended it to be also used on lcd or amoled screens.

thanks for the clarification on eink being static.

**Reply by Frenzie** [reply]
Yes, it has an LCD setting and it's on by default (concretely, color rendering and faster panning), but it definitely won't save you any energy. The basic way the program works is that nothing's happening unless you do something. Whether you like the mostly E Ink optimized interface is only a matter of taste.

**Reply by wa15** [reply]
thanks!, on 2nd thought maybe that battery usage was because of downloading dictionaries. I dont really like the interface, but i will get used to it. koreader displays my images correctly without extra blank pages so thats a major plus

**Reply by Zeyadas** [reply]
I read a lot using my android tablet (LCD screen) and I've tried tons of ebook reader apps, found 2 apps that are perfectly designed and satisfying, bookari premium and koreader. 
The bookari is never updated since 2018 I think. And now I use koreader exclusively. Very customizable and the developers respond always to fix any issues. And no, it doesn't consume battery power on my tablet.

**Reply by wa15** [reply]
They have updates on their blog outside if you really like them. I tried their old playstore version but I think I will go with koreader since it doesnt generate extra pages with the images in my epubs
